### DISCUSSION TITLE
grid specification string appears in output filename

### DISCUSSION THREAD

**Discussion by RogerHar** [dtcenter/METplus#2181] [Configuration] [answered]
I've set up regridding using a grid specification string in METplus:
`GRID_STAT_REGRID_TO_GRID = latlon 240 121 -90.0 0.0 1.5 1.5`

It's working fine, but the output filenames contain this entire string, e.g.:
`grid_stat_METO_Global_geopotential_height_vs_ANLYS_geopotential_height_latlon 240 121 -90.0 0.0 1.5 1.5_360000L_20180711_120000V_sl1l2.txt`
This includes spaces, which is a little inconvenient to deal with as you sometimes need to remember to prefix the spaces with backslashes.

Is there a way to reset the metadata(?) that ends up in this bit of the filename? I'd like to be able to give the grid a name of my choosing, e.g. "Global1.5x1.5degree". Or alternatively to replace the spaces with underscores.

This is only a minor inconvenience, if there isn't a way to do this it's not a big deal, I just wonder if I'm missing something.

**Comment by RogerHar** [comment, marked as answer]
Sorry I've just realised it's because my setting of  GRID_STAT_OUTPUT_PREFIX includes `..._{GRID_STAT_REGRID_TO_GRID}_...`, so I can change that to`... _Global1.5x1.5degree_...` . Apologies, one of my colleagues set up the conf originally and I changed the grid from a named one to a grid spec string.

I'll close this discussion - I'd delete it if I could!

**Reply by JohnHalleyGotway** [reply]
@RogerHar I do see an option to delete. If you think this discussion would be misleading for others, I can do that. Just let me know.
